config/default/kustomization.yaml

```diff
@@ -9,22 +9,12 @@ namespace: toolchain-member-operator
 namePrefix: member-operator-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+#commonLl someValue
 
-bases:
```

Comment:
renamed bases to resources because bases is now depricated

Reply:
I was wondering about that :) Is this the relevant issue you saw: https://github.com/kubernetes/kubectl/issues/891#issuecomment-731238572 Or did you see that somewhere else?